test/client/fixtures/utils/cookie-test.js

```diff
@@ -72,6 +72,7 @@ test('formatClientString', function () {
 });
 
 test('domainMatch', function () {
+    ok(cookieUtil.domainMatch('sub.example.com'));
```

Comment:
You call this function only with two arguments. So test it with two arguments as well: `ok(cookieUtil.domainMatch('sub.example.com', void 0))`.

For `pathMatch` too.